cloud/src/meta-service/meta_service.cpp

```diff
@@ -1328,6 +1328,12 @@ void MetaServiceImpl::prepare_restore_job(::google::protobuf::RpcController* con
                                           RestoreJobResponse* response,
                                           ::google::protobuf::Closure* done) {
     RPC_PREPROCESS(prepare_restore_job);
+    if (request->action() != RestoreJobRequest::PREPARE) {
+        code = MetaServiceCode::INVALID_ARGUMENT;
+        msg = "invalid action";
```

Comment:
the error msg should tell what action is expected  and what action is given

Reply:
pls check other occurrences

Reply:
done